astro/src/content/docs/get-started/core-concepts/roles.mdx

```diff
@@ -125,6 +125,7 @@ _FusionAuth application roles_
 | `user_manager`             | `631ecd9d-8d40-4c13-8277-80cedb8236f3` | Can add and edit users.<br/><br/>Please note that because this role can fully manage users, it is similar to `admin`. The `user_support_manager` is recommended in most cases.|
 | `user_support_manager`     | `631ecd9d-8d40-4c13-8277-80cedb823704` | Allows for a limited scope of user management. See below. <AvailableSince since="1.23.0"/>  |
 | `user_support_viewer`      | `631ecd9d-8d40-4c13-8277-80cedb823705` | Can view user information. <AvailableSince since="1.23.0"/>                        |
+| `webhook_event_log_viewer` | `631ecd9d-8d40-4c13-8277-80cedb823713` | Can view the webhook event log.                                                    |
```

Comment:
minor note: Intellij gives us the squigglies if the table isn't formatted evenly and it will fix it for you if you let it

Reply:
🤔 it's formatted evenly for me. IJ didn't give me the squiggles when I changed it.
![image](https://github.com/user-attachments/assets/f50366e8-073c-454c-9043-cd4017a4143b)

Reply:
noted. I am using the MDX plugin. I suppose if that should be consistent will need something that is more universal. Ignore for now.